android/build.gradle

```diff
@@ -55,9 +55,9 @@ android {
 
 dependencies {
     testImplementation 'junit:junit:4.13.2'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
-    implementation 'org.bouncycastle:bcprov-jdk15on:1.70'
-    implementation("com.squareup.okhttp3:okhttp:4.10.0")
-    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.3")
+    androidTestImplementation 'androidx.test.ext:junit:1.1.5'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
+    implementation 'org.bouncycastle:bcprov-jdk15to18:1.75'
```

Comment:
It can be either [bcprov-jdk18on](https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk18on) or [bcprov-jdk15to18](https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15to18) (currently selected)